[2025-03-06T06:28:40.052+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-06T06:28:40.056+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_horoscope_dag.get_combined_info scheduled__2025-03-05T06:17:53.686903+00:00 [queued]>
[2025-03-06T06:28:40.088+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_horoscope_dag.get_combined_info scheduled__2025-03-05T06:17:53.686903+00:00 [queued]>
[2025-03-06T06:28:40.088+0000] {taskinstance.py:1361} INFO - Starting attempt 3 of 4
[2025-03-06T06:28:40.094+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): get_combined_info> on 2025-03-05 06:17:53.686903+00:00
[2025-03-06T06:28:40.096+0000] {standard_task_runner.py:57} INFO - Started process 42 to run task
[2025-03-06T06:28:40.097+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_horoscope_dag', 'get_combined_info', 'scheduled__2025-03-05T06:17:53.686903+00:00', '--job-id', '38', '--raw', '--subdir', 'DAGS_FOLDER/email_dag.py', '--cfg-path', '/tmp/tmp4zj2gnci']
[2025-03-06T06:28:40.098+0000] {standard_task_runner.py:85} INFO - Job 38: Subtask get_combined_info
[2025-03-06T06:28:40.105+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-06T06:28:40.119+0000] {task_command.py:415} INFO - Running <TaskInstance: weather_horoscope_dag.get_combined_info scheduled__2025-03-05T06:17:53.686903+00:00 [running]> on host a6553d1b92df
[2025-03-06T06:28:40.124+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-06T06:28:40.146+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='liuzifeng_r@163.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_horoscope_dag' AIRFLOW_CTX_TASK_ID='get_combined_info' AIRFLOW_CTX_EXECUTION_DATE='2025-03-05T06:17:53.686903+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-05T06:17:53.686903+00:00'
